src/editor/svgedit.js

```diff
@@ -4640,6 +4626,10 @@ editor.init = () => {
     $id('tool_docprops').addEventListener('click', showDocProperties);
     $id('tool_editor_prefs').addEventListener('click', showPreferences);
     $id('tool_editor_homepage').addEventListener('click', openHomePage);
+    const seImgProp = document.getElementById('se-img-prop');
```

Comment:
I think it's better to move these event listener inside the code of the dialog. It's possible to create a custom event from the dialog that will send all values of the dialog to svgedit.js. svgedit will trap this event and update the different variables as needed.

(In general, I would prefer to avoid using the shadowRoot of a given element in svgedit.js)

In the web component dialog, you create a custom event with your values in under "detail":
```
const Event = new CustomEvent('change', {detail: {value1, value2, value3}, bubbles: false});
 this.dispatchEvent(Event);
```

In svgedit, you capture the custom event and you read the detail object:
const handleMyDialog = (e) =>{
    e.preventDefault();
    const {value1, value2, value3} = e.detail;
    ...do what ever is needed...
  };
}`````

 `$id('<<<<dialog id>>>> ').addEventListener('change', handleMyDialog);`